src/zzz_od/gui/view/one_dragon/inventory_scan_interface.py

```
@@ -0,0 +1,1184 @@
+                
+                # 更新代理人详细信息
+                agent_info = []
+                agent_info.append(f"{gt('代理人')}: {agent_name}")
+                agent_info.append(f"{gt('等级')}: {report.get('level', '未知')}")
+                agent_info.append(f"{gt('核心等级')}: {report.get('core', '未知')}")
+                agent_info.append(f"{gt('钥匙')}: {report.get('key', '未知')}")
+                
+                # 音擎信息
+                if 'equippedEngines' in report:
+                    agent_info.append(f"\n{gt('音擎')}:")
+                    for engine_key, engine in report['equippedEngines'].items():
```

Ruff (0.15.7)

[warning] Loop control variable `engine_key` not used within loop body

Rename unused `engine_key` to `_engine_key`

(B007)